tensorboard/uploader/uploader.py

```diff
@@ -720,6 +727,14 @@ def _get_or_create_blob_sequence(self):
     def _send_blob(self, blob_sequence_id, seq_index, blob):
```

Comment:
Could you please document the return value now that it’s non-`None`?
I can infer the semantics from clients but not from the function itself.

Reply:
Done.